### DISCUSSION TITLE
Kenapa menggunakan geom_line tapi hasilnya berubah menjadi geom_area?

### DISCUSSION THREAD

**Discussion by Nurfalah123** [teamalgoritma/community#95] [Q&A] [answered]
Izin bertanya, saat saya ingin menampilkan grafik line menggunakan geom_line namun setelah run chunck hasilnya seperti geom_area ya, kenapa ya?

Mohon bantuannya
![code selected](https://user-images.githubusercontent.com/103617918/167976397-957d1b3e-3738-4b1f-8444-55144216b849.PNG)
![coding plot](https://user-images.githubusercontent.com/103617918/167976424-9682c543-db99-4273-9c33-4b28f6ff690e.PNG)
![plot](https://user-images.githubusercontent.com/103617918/167976433-0c59999e-0f1b-48de-8df6-9cb861755bea.PNG)

**Comment by inytss** [comment, marked as answer]
Dear @Nurfalah123 ,
Sepertinya hal tersebut dikarenakan pada tanggal yang sama, terdapat beberapa provinsi yang akan ditampilkan sehingga terdapat beberapa line yang menumpuk. Ibu boleh mencoba melakukan filter terhadap provinsi tertentu terlebih dahulu lalu mencoba untuk membuat geom_line nya kembali.

**Reply by Nurfalah123** [reply]
baik saya coba dulu ya, terimakasih
